.changeset/cold-scissors-hunt.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/design-tokens': major
+'@twilio-paste/theme': major
+---
+
+[Design Tokens, Theme] Deprecate the legacy console theme. The default theme should be used instead.
```

Comment:
Do I need 2 separate changesets? Also, is this an adequate message or do I need to add more information?

Reply:
Also does core need to be included?

Reply:
Core must be included. 

Theme is technically a minor as it's not breaking. You don't need to do anything.

Design tokens is a major.

Also consider the message you would want to read from the changelog about this? What would you want to know, how would you want to be prepared for it as a consumer.

Reply:
It looks like the theme package has a breaking change too, check out https://github.com/twilio-labs/paste/pull/2168/files#diff-086a3fa08d1651562b139d92ed171a8e1e9c5af00da5988e754eb3a7d4b7edc7R15-R17

I left a comment on how to resolve the breaking change

Reply:
Fixed the breaking changes!